platform/swift/source/LoggerBridge.swift

```diff
@@ -30,6 +55,14 @@ final class LoggerBridge: LoggerBridging {
         network: Network?,
         errorReporting: RemoteErrorReporting
     ) {
+        do {
+            try bufferDirectoryPath.map(makeDirectoryAndDisableProtection(at:))
+        } catch {
+            // To be safe we don't initialize the logger if we can't create the directory or set
+            // the file protection policy.
+            return nil
```

Comment:
Should we warn here or something so it's obvious at least in local dev if this fails for some reason?

Reply:
I'll follow up on a diff PR, the reason I didn't do that is we don't have a nice way of logging stuff right now and there are more cases where we return `nil` silently